core/block.js

```diff
@@ -1217,15 +1217,30 @@ Blockly.Block.prototype.getInputsInline = function() {
 /**
  * Set whether the block is disabled or not.
  * @param {boolean} disabled True if disabled.
+ * @deprecated April 2018
  */
 Blockly.Block.prototype.setDisabled = function(disabled) {
+  console.warn('Deprecated call to Blockly.Block.prototype.setDisabled, ' +
+               'use Blockly.Block.prototype.setEnabled instead.');
   if (this.disabled != disabled) {
```

Comment:
This should call setEnabled(!disabled) instead so we don't have duplicate implementations.